es-core/src/components/GridTileComponent.h

```diff
@@ -0,0 +1,97 @@
+#include "GuiComponent.h"
+#include "components/ComponentGrid.h"
+#include "components/NinePatchComponent.h"
+#include "components/TextComponent.h"
+#include "components/ImageComponent.h"
+
+class TextComponent;
+class ImageComponent;
+
+const Eigen::Vector2f NULLVECT = { 0,0 };
+
+struct KeyframeVariables {
+	Eigen::Vector2f size;
+	Eigen::Vector2f pos;
+	Eigen::Vector2f backgroundSize;
+	Eigen::Vector2f textContainerSize;
+	unsigned char opacity = 0xFF;
+	unsigned int color = 0xFFFFFFFF;
+	unsigned int backgroundColor = 0xFFFFFFFF;
+	unsigned int textColor = 0xFFFFFFFF;
+};
+
+struct CursorAnimation {
```

Comment:
This is called CursorAnimation because the only animated tile is the one selected by the cursor (and the one which just got unselected when you move the cursor). Should I still change that ?

Reply:
Thanks, I now understand this is only the subset of animations as a result of cursor changes, not the scrolling of the grid itself etc. Your call if you want to keep the name.